### PR TITLE
Fix small typo in transformation docs

### LINKED ISSUE #1277: Typo in RectangularBoundariesTransformation docs
https://github.com/pints-team/pints/blob/a6a1a39c322c8419ff929b82f73ed3f7aa8b72ef/pints/_transformation.py#L708-L723

### PR DESCRIPTION
**Pull request by chonlei**
Fix #1277

### CODE REVIEW

File: pints/_transformation.py [hunk 1]
```diff
@@ -708,13 +708,13 @@ def to_search(self, p):
 class RectangularBoundariesTransformation(Transformation):
     r"""
     A generalised version of the logit transformation for the model parameters,
-    which transform an interval or rectangular boundaries :math:`[a, b)` to
+    which transform an interval or rectangular boundaries :math:`(a, b)` to
```
Comment (MichaelClerx) [inline pints/_transformation.py:711]: transform -> transforms.

Is it a () interval then, not a half-open one?

**Comment by MichaelClerx** [comment]
Tests are failing. I see Ben has a fix here: https://github.com/pints-team/pints/pull/1058/files

Maybe stick this fix in this PR too? Makes more sense here :D

File: pints/_transformation.py (hunk 1 above)
Comment (chonlei) [reply pints/_transformation.py:711]: () is an interval, yes.

**Comment by chonlei** [comment]
Michael did you mean stick this fix (https://github.com/pints-team/pints/pull/1058/files#diff-43f514bb09c1eb58423de94253a01bf25b3232f4324e4a9d8606ee6db00039b5R160) to here?

File: pints/_transformation.py (hunk 1 above)
Comment (MichaelClerx) [reply pints/_transformation.py:711]: But you're saying the interval doesn't include `a` then? So it's an open rather than a half-open interval?

**Comment by MichaelClerx** [comment]
> Michael did you mean stick this fix (https://github.com/pints-team/pints/pull/1058/files#diff-43f514bb09c1eb58423de94253a01bf25b3232f4324e4a9d8606ee6db00039b5R160) to here?

Yes please!

File: pints/_transformation.py (hunk 1 above)
Comment (chonlei) [reply pints/_transformation.py]: Ha, need correcting that line too! :smile:

Comment (chonlei) [reply pints/_transformation.py]: It's an open () interval. I think now the docs is consistent...?

Comment (MichaelClerx) [reply pints/_transformation.py]: :D Are you sure? The rectangular boundaries docs say they define a half-open interval https://pints.readthedocs.io/en/stable/boundaries.html#pints.RectangularBoundaries

Comment (chonlei) [reply pints/_transformation.py]: Oh... didn't realise that!

Comment (chonlei) [reply pints/_transformation.py]: I can't remember the reason - why do we want to include the lower bound?

Comment (MichaelClerx) [reply pints/_transformation.py]: Not sure, but maybe a separate ticket then? This can just fix the missing "s" and test :D :D :D

Comment (chonlei) [reply pints/_transformation.py]: See #1279.

**Review by MichaelClerx — APPROVED** [review]

**Pull request merged**